Makefile

```diff
@@ -0,0 +1,23 @@
+GUI_OPTIONS=--env=DISPLAY --env=QT_X11_NO_MITSHM=1 -v /tmp/.X11-unix:/tmp/.X11-unix
+GENERAL_OPTIONS=-it --rm --privileged --net=host --volume $(PWD)/software:/software
```

Comment:
Maybe you'd like to assign the volume to a more representative name, like catkin_ws or workspace. Up to you.